masonry/src/widget/text_area.rs

```diff
@@ -720,12 +767,15 @@ impl<const EDITABLE: bool> Widget for TextArea<EDITABLE> {
     }
 
     fn accepts_text_input(&self) -> bool {
-        // TODO: Implement IME, then flip back to EDITABLE.
-        false
+        EDITABLE
     }
 
     fn on_access_event(&mut self, ctx: &mut EventCtx, event: &AccessEvent) {
         if event.action == accesskit::Action::SetTextSelection {
+            if self.editor.is_composing() {
+                return;
+            }
```

Comment:
We should check with @mwcampbell if this is right. The scenario is:
1) The user is inputting some text which uses a pre-edit through their IME
2) Through their accessibility tool, they have tried to select some text, without their IME finishing the input

What should we do in that case? This code just ignores the second request, which is the easiest method to reason about. The alternatives are quite messy.